lib/content-services/src/lib/category/categories-management/categories-management.component.scss

```diff
@@ -11,6 +17,20 @@
         .adf-btn-padded {
             margin-right: -14px;
         }
+
+        .mat-form-field-underline {
+            display: none;
+        }
+
+        .mat-form-field-appearance-fill {
+            .mat-form-field-flex {
+                background: none;
```

Comment:
what are you trying to achieve?

Reply:
Without using a background display none, here is a screenshot:
![background_none](https://github.com/Alfresco/alfresco-ng2-components/assets/107834418/41a4b439-357f-42c0-b1ff-cbfea2e853d6)
The style was added to achieve the look as shown in Figma

Reply:
got it